src/calendar-tile.js

```diff
@@ -49,7 +49,7 @@ class CalendarTile extends Component {
 		const overOneWeek = this.isOverOneWeek(startMoment)
 
 		this.setState({
-			startDate: !overOneWeek ? DAYS_OF_WEEK[startMoment.day()] : startMoment.format('MMM Do'),
+			startDate: !overOneWeek ? DAYS_OF_WEEK[startMoment.day()+1] : startMoment.format('MMM Do'),
```

Comment:
I agree with Erich. Here's a link that might be helpful: https://momentjs.com/docs/#/get-set/day/